flair/data.py

```diff
@@ -588,7 +588,12 @@ def __copy__(self):
         return s
 
     def __str__(self) -> str:
-        return 'Sentence: "{}" - {} Tokens'.format(' '.join([t.text for t in self.tokens]), len(self))
+
+        if len(self.labels) == 0:
```

Comment:
Don't you want to avoid calculating length and use implicit boolean conversion? `if not self.labels` ?

Reply:
Thanks - just added this!